### PR TITLE
acrn-config: UI supports to edit mulitple virtio input devices

### PR DESCRIPTION
**Pull request by szhen11**
This is to make acrn config UI supports to edit more than one virtio
input devices in multi-lines

Tracked-On: #4163
Signed-off-by: Shuang Zheng <shuang.zheng@intel.com>
Reviewed-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by szhen11** [comment]
> @szhen11 Could you rephrase the commit message to describe the change clearer? What does "edited by multi-lines" mean?
Hi, @binbinwu1 the commit message is updated.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>
